app/js/views/services.jsx

```diff
@@ -65,25 +63,15 @@ export default class Services extends React.Component {
     this.setState({ services });
   }
 
-  dismissModal() {
-    this.setState({ isModalVisible: false });
+  renderHeader() {
+    return super.renderHeader('My Home');
   }
 
-  render() {
+  renderBody() {
     return (
-      <div className="app-view">
-        <header className="app-view__header">
-          <h1>My Home</h1>
-        </header>
-        <section className="app-view__body">
-          <h2>General</h2>
-          <ServicesList services={this.state.services} foxbox={this.foxbox}/>
-        </section>
-        <footer className="app-view__footer">
-          <NavigationMenu foxbox={this.foxbox}/>
-        </footer>
-        <Modal visible={this.state.isModalVisible} title={this.state.title}
-               body={this.state.body} dismiss={this.dismissModal.bind(this)}/>
+      <div className="app-view__body-placeholder">
```

Comment:
Do we really need the `app-view__body-placeholder` class here? It looks like it's going to be something easy to forget if we want to add more top views.